sequencer/src/state_signature.rs

```diff
@@ -45,18 +45,14 @@ pub struct StateSigner<ApiVer: StaticVersionType> {
     /// The most recent light client state signatures
     signatures: RwLock<StateSignatureMemStorage>,
 
-    /// Commitment for current fixed stake table
-    stake_table_comm: StakeTableCommitmentType,
-
```

Comment:
indeed, we are not updating this anymore, but I also don't see any harm of keeping it (as a static member field of this struct